pkg/apis/edge/v1alpha1/edge-placement.go

```diff
@@ -20,14 +20,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EdgePlacement binds a collection of (a) Namespaces and non-namespaced objects
-// to (b) a collection of Locations.
-// An EdgePlacement object appears in the center and directs all the syncable
-// objects in the selected Namespaces and all the selected non-namespaced objects
-// to propagate to _all_ of the selected Locations
-// (one SyncTarget in each such Location).
-// This is not entirely unrelated to a TMC Placement, which directs the selected
-// Namespaces to propagate to _one_ of the selected Locations.
+// EdgePlacement exists in the center and binds (a) a collection of
+// Locations with (b) both (b1) objects in the center to downsync
+// (propagate desired state from center to edge and return reported
+// state from edge toward center), and (b2) a way of identifying objects
+// (in edge clusters) to upsync (propagate from edge toward center).
+// Both downsyncing and upsyncing are with all of the Locations.  This
```

Comment:
@MikeSpreitzer Is the set of Locations selected only by label of the Location object? Can we assume someone who defines this edge placement spec has knowledge on how to pick label for selecting destination?

Reply:
Yes and Yes.
This should be described more clearly in the outline.
It differs from TMC, I think, although my understanding of TMC is incomplete.  As far as I know, in TMC the Placement has a predicate over SyncTarget labels but the SyncTarget objects are not intended to be exposed to the authors of Placements.